Makefile

```
@@ -44,7 +44,7 @@ endif
 
 LDFLAGS=
 ifdef STATIC
-	export CGO_ENABLED=0
+	export CGO_ENABLED=1
```

@vrindle  I don't think we should do this. This will diverge from U/S

https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/0bf8f4fa9dc4ef6a0b88c875142815676d3bf311/Makefile#L46

---

I think we need it..

---

Why are we even setting `CGO_ENABLED` twice?

---

What about the following in upstream. We can then set FIPS_COMPLIANT when we build downstream specifically.

```suggestion
ifdef STATIC
    ifndef FIPS_COMPLIANT
        export CGO_ENABLED=0
        EXTLDFLAGS=-extldflags "-static"
    else
        $(error STATIC and FIPS_COMPLIANT cannot be set simultaneously)
    endif
else
    export CGO_ENABLED=1
    EXTLDFLAGS=-extldflags ""
endif

LDFLAGS=-a -ldflags '$(EXTLDFLAGS)'
```

also, I don't think we need to set CGO_ENABLED twice, i.e. we export it first, so why specify it on line 74 again? 

I agree with removing `-tags no_openssl` in general. Is there a need to ever have it in the first place?